_posts/2020-03-31-Ordina-Young-Professional-Programme-2020-Light-Edition.md

```diff
@@ -0,0 +1,206 @@
+which is one of the must reads for every programmer.
+To be able to write clean code, an understanding of what bad code is is needed. 
+This is also explained in the first chapter of the book.
+Some examples of topics in the book we still use everyday are: 
+- Avoiding comments but using descriptive names
+- Functions should be small and do one thing
+- Use one word per concept
+- ...
+
+# Dev case
+### Introduction
+For our dev case, our task was to make a fully functional web application for MFC Combo. 
+MFC Combo is an organization in youth care who guide families and children with a difficult situation at home. 
+They use the "Combobox" in their meetings with the families. 
+This is used to make communication easier and more fun.
+right medium now, they have physical stickers and papers to let the child express their feelings, 
+but they want it to become digital so that it's easier to follow-up a child.
+
+## Technology & Methodologies
+We worked Agile in two-weekly sprints with all the Scrum Ceremonies. 
+Azure DevOps made this process easy and made sure our code was always clean and working as expected using CI/CD. 
+For the front end, we used the Angular Framework to develop the application, combined with Jasmine and Karma for testing. Authentication was done using AWS Amplify, which is a development platform. 
+This allowed us to quickly set up the authentication and authorization of our app. 
+Sentry.io was used for front end logging and monitoring after deployment. 
+This way, we had full control over our code, from development to deployment and even after deployment.
+For the back end, we used Java Spring Boot 2.3.1. 
+Our Database was a noSQL DynamoDB, for user management we used Cognito and finally for our file storage we used an S3 bucket. All of these services were set up in AWS. 
+This means we used the AWS Java SDK to manipulate these services in an easy and structured way.
+
+### Result
+<img class="image" alt="Drag & drop gif"  style="max-width: 50%; padding-left medium: 4px" src="/img/2020-03-31-Young-Professional-2020-Light/board.gif">
+
+After 10 sprints we created an online platform for Combobox. 
+This platform would assist them in their day to day operations. 
+The application included various features like: drag & drop capabilities, user management, file management, 
+sticker management, overview of current cases, authorization & authentication. 
+One of the most challenging parts of the project was the implementation of the Drag & Drop feature, 
+for which we created a new directive. 
+The result can be seen in the following screenshots:
+
+<img class="image" alt="Login Page" style="max-width: 100%;" src="/img/2020-03-31-Young-Professional-2020-Light/login.jpg">
+
+<img class="image" alt="Upload Page"  style="max-width: 100%;" src="/img/2020-03-31-Young-Professional-2020-Light/upload.jpg">
+
+<img class="image" alt="Fox Icon" src="/img/2020-03-31-Young-Professional-2020-Light/icons8-fox-48.png"
```

Comment:
What's this? 🤓

Reply:
Tribute to our dev case coach 😁  (if it's okay 😅 )